persist-cli/src/main/java/io/ballerina/persist/cmd/Push.java

```diff
@@ -96,17 +109,27 @@ public void execute() {
             errStream.println(commandUsageInfo);
             return;
         }
+        Path persistTomlPath = Paths.get(this.sourcePath, SUBMODULE_PERSIST, PERSIST_TOML_FILE);
+        Path databaseConfigurationBalPath = Paths.get(this.sourcePath, SUBMODULE_MODULES, KEYWORD_CLIENTS,
```

Comment:
```suggestion
        Path databaseConfigurationBalPath = Paths.get(this.sourcePath, SUBMODULE_FOLDER, KEYWORD_CLIENTS,
```

Reply:
Updated in [f568f12](https://github.com/ballerina-platform/persist-tools/pull/53/commits/f568f12dead6d22dfecd502349fd2ff4b86395f1)